services/web-app/components/footer/footer.js

```diff
@@ -4,71 +4,104 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
-import { Column, Grid } from '@carbon/react'
-import { IbmCloud } from '@carbon/react/icons'
+import { Column, Grid, Theme } from '@carbon/react'
 import clsx from 'clsx'
 import Link from 'next/link'
 import PropTypes from 'prop-types'
 
 import styles from './footer.module.scss'
 
-const Footer = () => {
+const Footer = ({ hasSideNav }) => {
+  const colOne = [
+    {
+      text: 'Contribute',
+      link: 'https://github.com/carbon-design-system/carbon-platform'
+    },
+    {
+      text: 'Privacy',
+      link: 'https://github.com/carbon-design-system/carbon-platform'
+    },
+    {
+      text: 'Terms of Use',
+      link: 'https://github.com/carbon-design-system/carbon-platform'
+    },
+    {
+      text: 'IBM.com',
+      link: 'https://github.com/carbon-design-system/carbon-platform'
```

Comment:
```suggestion
      link: 'https://www.ibm.com/'
```
Figured we could add in some of these links?